src/jinja2/nodes.py

```diff
@@ -339,7 +339,7 @@ class With(Stmt):
 class Block(Stmt):
     """A node that represents a block."""
 
-    fields = ("name", "body", "scoped")
+    fields = ("name", "body", "scoped", "required")
```

Comment:
Add a ``versionchanged`` entry in the docstring to mention this.

Reply:
👍 added it in as version 3.0.0